src/util/Serializer/TripleSerializer.h

```diff
@@ -67,21 +68,36 @@ CPP_template(typename Serializer)(
     requires serialization::WriteSerializer<
         Serializer>) void serializeLocalVocab(Serializer& serializer,
                                               const LocalVocab& vocab) {
-  AD_CONTRACT_CHECK(vocab.numSets() == 1);
-  const auto& words = vocab.primaryWordSet();
-  serializer << words.size();
-  ql::ranges::for_each(words, [&serializer](const auto& localVocabEntry) {
-    serializer << Id::makeFromLocalVocabIndex(&localVocabEntry);
-    serializer << localVocabEntry.toStringRepresentation();
-  });
+  serializer << vocab.getOwnedLocalBlankNodeBlocks();
```

Comment:
This breaks the persisted update triples.... Make sure that this is documented somewhere.